modern/src/common/components/StatusCard.js

```diff
@@ -57,6 +57,8 @@ const useStyles = makeStyles((theme) => ({
   content: {
     paddingTop: theme.spacing(1),
     paddingBottom: theme.spacing(1),
+    'max-height': '40vh',
```

Comment:
You have to use camelcase name here. And also the constant should be in the same place where we put other similar constants. It's part of the theme.